src/php/aspell.php

```diff
@@ -60,6 +60,9 @@
     // when splitting words.
     $text = preg_replace('/--+/u', ' ', $text);
 
+    // Get rid of anything containing @ : # ~ or / (hashtags, email addresses, URLs, etc.)
+    $text = preg_replace('/[\S+)]*[@\/:#~][\S+)]*/', '', $text);
```

Comment:
I'm a little unclear on why the '+' and ')' are in the regex.  Am I missing something? I was thinking a slightly simpler regex may suffice (note I also added in \\ for windows style path names):
```
    [\S]*[@\/\\:#~][\S]*
```